packages/web-components/src/components.d.ts

```diff
@@ -166,6 +166,41 @@ export namespace Components {
          */
         "setFocus": boolean;
     }
+    interface VaModal {
+        /**
+          * Click outside modal will trigger closeEvent
+         */
+        "clickToClose"?: boolean;
+        /**
+          * If true, analytics event won't be fired
+         */
+        "disableAnalytics"?: boolean;
+        /**
+          * Hide the close button that's normally in the top right
+         */
+        "hideCloseButton"?: boolean;
+        /**
+          * Selector to explicitly specify which element should receive focus when the modal is open, if the initially focused element is not the first focusable element in the document
+         */
+        "initialFocusSelector"?: string;
+        /**
+          * Title/header text for the modal
+         */
+        "modalTitle"?: string;
+        /**
+          * Primary button text and action
+         */
+        "primaryButton"?: { text: string; action: () => void };
+        /**
+          * Secondary button text and action
+         */
+        "secondaryButton"?: { text: string; action: () => void };
+        "status"?: 'continue' | 'error' | 'info' | 'success' | 'warning';
```

Comment:
@andresriveratoro missing the comment here and then looking at storybook we are getting an `undefined` prop being set below `visible`: https://60f9b557105290003b387cd5-dighdkhdhk.chromatic.com/?path=/docs/components-va-modal--default

Reply:
I can fix this by updating the type to `any` . I haven't been able to figure out why this happens in Storybook when specifying object or function types.

Reply:
Yeah its weird because the whole "Style of modal alert - info, error, success, warning" seems to get eaten during compilation